src/sentry/api/endpoints/check_am2_compatibility.py

```diff
@@ -16,7 +16,7 @@
 
 
 # NOTE: This endpoint should be in getsentry
-@region_silo_endpoint
+@all_silo_endpoint
```

Comment:
This should stay as a region_silo_endpoint because of the resources it uses. You can either route requests from the frontend to the customer's region URL, or add this endpoint to `REGION_PINNED_URL_NAMES`.

I recommend routing via the frontend. Each `Organization` in the client app will have `organization.links.regionUrl` set and you can use that to send your request to the correct region.

Reply:
I'll use the `regionUrl`! Thanks again